### DISCUSSION TITLE
Can't use Tailwind's values for margins, paddings etc.

### DISCUSSION THREAD

**Discussion by dm-sheremetiev** [tailwindlabs/tailwindcss#13392] [Help] [answered]
Hi guys. For some reason I can't use classes like mt-4, py-10. I can only use determined values like mt-[16px] py-[40px] and it's annoying. I don't know why because on my work's projects I don't have any problem. 

There is my tailwind config file. Maybe someone also had this problem or we have issue with this. Thanks!

NOTE: I'm using antd for some components but it shouldn't effect because for these types of values any styles were implemented.

```
/** @type {import('tailwindcss').Config} */
module.exports = {
  content: [
    './src/**/*.{js,ts,jsx,tsx}',
    './app/**/*.{js,ts,jsx,tsx,mdx}',
    './pages/**/*.{js,ts,jsx,tsx,mdx}',
    './components/**/*.{js,ts,jsx,tsx,mdx}',

    './src/**/*.{js,ts,jsx,tsx,mdx}',
  ],

  theme: {
    colors: {
      transparent: 'transparent',
      'love-fumes': '#FFD1D3',
      'fluorescent-red': '#FD5059',
      'dr-white': '#FBFAF9',
      'kokushoku-black': '#171411',
      'soft-pink': '#FBB4BF',
      'infrared-burn': '#DB2C35',
      'hadfield-blue': '#1677ff',
      'dead-blue-eyes': '#025ac5',
      'silver-surfer': '#7C7C8A',
    },
    screens: {
      sm: '640px',
      // => @media (min-width: 640px) { ... }

      md: '768px',
      // => @media (min-width: 768px) { ... }

      lg: '1024px',
      // => @media (min-width: 1024px) { ... }

      xl: '1280px',
      // => @media (min-width: 1280px) { ... }

      '2xl': '1536px',
      // => @media (min-width: 1536px) { ... }
    },
    spacing: {
      sm: '8px',
      md: '16px',
      lg: '24px',
      xl: '64px',
    },
    extend: {
      screens: {
        xsm: '300',
        // => @media (min-width: 300px) { ... }
      },
      fontFamily: {
        comforta: ['Comfortaa', 'sans-serif'],
        artegra: ['ArtegraSans', 'sans-serif'],
      },
    },
  },
  plugins: [],
};

```

**Comment by savicaleksa** [comment, marked as answer]
Try moving spacings object inside of extend object, I think you've accidentally replaced all the default spacings with those you've specified.

**Reply by dm-sheremetiev** [reply]
Thank you a lot. For some reason colors also should be placed into extends because if I left it how I had it before colors like bg-black also don't work (maybe I lost something in documentation). Anyway, thank you again.

**Reply by savicaleksa** [reply]
You can learn more about extending the default theme [here](https://tailwindcss.com/docs/theme#extending-the-default-theme)
